t/01-basic.t

```diff
@@ -9,10 +9,11 @@ my @res;
 my @l = 'a'..'z';
 my $now = now;
 my $first-response;
+my $idx = 0;
 react whenever proc-q
     @l.map({
         $*EXECUTABLE, '-e',
-        "say '$_' ~ \$*IN.slurp; note '$_'; sleep {2*($++/5).Int}; exit {$++}"
+        "say '$_' ~ \$*IN.slurp; note '$_'; sleep {2*($idx/5).Int}; exit {$idx++}"
```

Comment:
I rather the original be changed to: 

```perl6
"say '$_' ~ \$*IN.slurp; note '$_'; sleep $(2*($++/5).Int); exit $($++)"
```

Instead of using a single variable with unpredictable increment, like that.

Reply:
for whatever reason that doesn’t work on my box with rak 2018.02.1, and i’m testing it with prove6